src/components/Switch/switch.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './switch.component.css';
+
+const Switch = () => {
+  let [checked, setChecked] = useState(false);
+
+  function toggle() {
+    setChecked((prevState) => !prevState);
+  }
+
+  return (
+    <label className="inputWrapper">
+      <input className="input" type="checkbox" onChange={toggle}></input>

```

Comment:
Technique tip:

```suggestion
  let [checked, setChecked] = useState(false);

  return (
    <label className="inputWrapper">
      <input className="input" type="checkbox" onChange={() => setChecked(!checked)}></input>
      ...
```

You can avoid the need for a middleman function declaration.